Tools/changelog/package.json

```diff
@@ -1,5 +1,7 @@
 {
     "name": "changelogs",
+    "author": "DEATHB4DEFEAT",
+    "license": "MIT",
```

Comment:
Is this okay to have in an AGPL codebase?

Reply:
We have WizDen in here, of course.

Reply:
Yeah, but that naturally got relicensed no? Idrk how this business works

Reply:
Same thing here, you can include MIT code in AGPL codebases